diff --git a/modifiers.d.ts b/modifiers.d.ts
index 62bf3bccfc29ac73f50f613bb13489751745e77d..21f99a53589fc4a0b88f55f92803036c45160a9d 100644
--- a/modifiers.d.ts
+++ b/modifiers.d.ts
@@ -36,7 +36,7 @@ declare class AxisModifier extends Modifier<DragDropManager<any, any>, Options$1
      * @param options - The axis restriction options
      * @returns A configured AxisModifier instance
      */
-    static configure: (options: Options$1) => _dnd_kit_abstract.PluginDescriptor<any, any, typeof AxisModifier>;
+    static configure: (options: Options$1) => _dnd_kit_abstract.PluginDescriptor<any, any, any>;
 }
 /**
  * A pre-configured modifier that restricts movement to the vertical axis.
@@ -44,14 +44,14 @@ declare class AxisModifier extends Modifier<DragDropManager<any, any>, Options$1
  * @remarks
  * This modifier fixes the x-axis value to 0, allowing only vertical movement.
  */
-declare const RestrictToVerticalAxis: _dnd_kit_abstract.PluginDescriptor<any, any, typeof AxisModifier>;
+declare const RestrictToVerticalAxis: _dnd_kit_abstract.PluginDescriptor<any, any, any>;
 /**
  * A pre-configured modifier that restricts movement to the horizontal axis.
  *
  * @remarks
  * This modifier fixes the y-axis value to 0, allowing only horizontal movement.
  */
-declare const RestrictToHorizontalAxis: _dnd_kit_abstract.PluginDescriptor<any, any, typeof AxisModifier>;
+declare const RestrictToHorizontalAxis: _dnd_kit_abstract.PluginDescriptor<any, any, any>;
 
 /**
  * Restricts a shape's movement to stay within a bounding rectangle.
@@ -128,7 +128,7 @@ declare class SnapModifier extends Modifier<DragDropManager<any, any>, Options>
      * @param options - The snap grid options
      * @returns A configured SnapModifier instance
      */
-    static configure: (options: Options) => _dnd_kit_abstract.PluginDescriptor<any, any, typeof SnapModifier>;
+    static configure: (options: Options) => _dnd_kit_abstract.PluginDescriptor<any, any, any>;
 }
 
 export { AxisModifier, RestrictToHorizontalAxis, RestrictToVerticalAxis, SnapModifier, restrictShapeToBoundingRectangle };
