diff --git a/index.js b/index.js
index ac8033b..b1ba8f0 100644
--- a/index.js
+++ b/index.js
@@ -21,9 +21,10 @@ class FixedJSDOMEnvironment extends JSDOMEnvironment {
     this.global.TextEncoderStream = TextEncoderStream
     this.global.ReadableStream = ReadableStream

-    this.global.Blob = Blob
+    // this.global.Blob = Blob
+    // this.global.File = File
     this.global.Headers = Headers
-    this.global.FormData = FormData
+    // this.global.FormData = FormData
     this.global.Request = Request
     this.global.Response = Response
     this.global.fetch = fetch
