diff --git a/dist/vis_annotation-DrDDbCAb.mjs b/dist/vis_annotation-DrDDbCAb.mjs
index 7298afd96a6195508c85ca0360961f30079f6a1d..7618291b76414d3632b346fcc6ba06f6ec935586 100644
--- a/dist/vis_annotation-DrDDbCAb.mjs
+++ b/dist/vis_annotation-DrDDbCAb.mjs
@@ -23,7 +23,7 @@ function toMatchImageSnapshot$1(options) {
 	/* v8 ignore start: stub as success when not in a test (e.g. in a story) */
 	if (!test) return Promise.resolve();
 	/* v8 ignore end */
-	return matchPageImageSnapshotAction(commands, toTaskId(test), options);
+	return matchPageImageSnapshotAction(commands, toTaskId(test), options, test);
 }
 
 //#endregion
@@ -87,7 +87,7 @@ function toMatchImageSnapshot(subject, options) {
 	/* v8 ignore start: stub as success when not in a test (e.g. in a story) */
 	if (!test) return Promise.resolve(success);
 	/* v8 ignore end */
-	return matchImageSnapshotAction(commands, toTaskId(test), subject, options).then(() => success);
+	return matchImageSnapshotAction(commands, toTaskId(test), subject, options, test).then(() => success);
 }
 
 //#endregion
