diff --git a/dist/vis_annotation-BVGOD6dC.mjs b/dist/vis_annotation-BVGOD6dC.mjs
index 484cdefd007f1075aaa1544a568ec458fb8b0049..c29b698c42b0b900a7b07c33dbdb76da204f7a1b 100644
--- a/dist/vis_annotation-BVGOD6dC.mjs
+++ b/dist/vis_annotation-BVGOD6dC.mjs
@@ -9,7 +9,7 @@ function toMatchImageSnapshot$1(options) {
 	/* v8 ignore start: stub as success when not in a test (e.g. in a story) */
 	if (!test) return Promise.resolve();
 	/* v8 ignore end */
-	return matchPageImageSnapshotAction(commands, toTaskId(test), options);
+	return matchPageImageSnapshotAction(commands, toTaskId(test), options, test);
 }
 
 //#endregion
@@ -42,7 +42,7 @@ function toMatchImageSnapshot(subject, options) {
 	/* v8 ignore start: stub as success when not in a test (e.g. in a story) */
 	if (!test) return Promise.resolve(success);
 	/* v8 ignore end */
-	return matchImageSnapshotAction(commands, toTaskId(test), subject, options).then(() => success);
+	return matchImageSnapshotAction(commands, toTaskId(test), subject, options, test).then(() => success);
 }
 
 //#endregion
